portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,159 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
 }
 
 #greeting-container {
+  margin-top: 3px;
+  color:white;
+  size: 20px;
+  background: gray; 
+}
+
+.image-thumbnails {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.image-thumbnails img {
+  margin: 10px;
+  max-height: 200px;
+  max-width: 200px;
+}
+
+.image-thumbnails img:hover {`
+  border: thick solid blue;
+}
+
+#random-image-container img {
   margin-top: 20px;
 }
+
+body {
+  font-family: Arial;
+  background-image:linear-gradient(rgba(0,0,0,0.2),rgba(0,0,0,0.2)),url(images/back.jpg);
+}
+
+.wrapper {
+  display: grid;
+    grid-template-columns: 1000px 1000px 1080px;
+  grid-auto-rows: 200px;
+}
+*
+{
```

Comment:
nit: Here and throughout this file, let's keep the selector and "{" on the same line. So your properties read like:

.example-class-name {
 // Some CSS
}

instead of

.example-class-name
{
 // Some CSS
}

Reply:
fixed